app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -82,11 +82,6 @@ def prepare_for_evss!
 
     save_metadata(forward_to_mas_all_claims: true) if Flipper.enabled?(:rrd_mas_all_claims_tracking) &&
                                                       !single_issue_hypertension_cfi?
-
-    if Flipper.enabled?(:rrd_mas_disability_tracking) && single_disability_eligible_for_mas?
-      save_metadata(forward_to_mas: true)
```

Comment:
This is the only place where the `forward_to_mas` metadata flag gets set. Removing this if-block lets us remove related `forward_to_mas` code below in the `conditionally_notify_mas` method.